base/char.jl

```diff
@@ -16,15 +18,14 @@ The [`isvalid`](@ref) function can be used to check which codepoints are
 representable in a given `AbstractChar` type.
 
 Internally, an `AbstractChar` type may use a variety of encodings.  Conversion
-to `UInt32` will not reveal this encoding because it always returns the
-Unicode value of the character.  (Typically, the raw encoding can be obtained
-via [`reinterpret`](@ref).)
-
-`print(io, c)` of any `c::AbstractChar` produces UTF-8 output by default
-(via conversion to `Char` if necessary).   `write(io, c)`, in contrast,
-may emit a different encoding depending on `typeof(c)`, and `read(io, typeof(c))`
-should read the same encoding as `write`.  New `AbstractChar` types should provide
-their own implementations of `write` and `read`.
+via `codepoint(char)` will not reveal this encoding because it always returns the
+Unicode value of the character. `print(io, c)` of any `c::AbstractChar`
+produces UTF-8 output by default (via conversion to `Char` if necessary).
```

Comment:
Perhaps clarify that it's not necessary that `print(io, c)` produces UTF-8 but rather that the output encoding is determined by `io` – and the built-in `IO` types are all UTF-8.